janus-idp/operator
# chore(deps): upgrade operator deps and renovate config to use go 1.21 and ubi 9.4

## Description
Upgrade the operator to use Go 1.21 and UBI 9.4 to ensure the resulting image is A grade.

## Which issue(s) does this PR fix or relate to
- Fixes #[RHIDP-2646](https://issues.redhat.com//browse/RHIDP-2646) - https://issues.redhat.com/browse/RHIDP-2646 

## PR acceptance criteria

## How to test changes / Special notes to the reviewer
1. Verify the build works by running `make image-build`.
2. Ensure all tests pass by running `make test`.
3. Review the changes to `go.mod`, `Dockerfile`, and renovate config.
